Mlem/App/Views/Root/TransitionView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  AccountTransitionView.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2023-10-02.
+//
+
+import Foundation
+import SwiftUI
+
+struct TransitionView: View {
+    let accountName: String?
+    @State var accountNameOpacity: CGFloat = .zero
+    
+    var body: some View {
+        VStack(spacing: 24) {
+            if let accountName {
+                Text("Welcome")
```

Comment:
Guest accounts displaying "Welcome lemmy.world" is slightly strange--it would make more sense for them to display something like "Welcome to lemmy.world"